src/Task1.kt

```diff
@@ -0,0 +1,12 @@
+
+fun IntArray.sumOfEven():Int {
+    var sum = 0
+    this.forEach{
```

Comment:
We can do better. Can you write this as a 1 liner ?

Reply:
You mean like the [first implementation ](https://github.com/Agilefreaks/Test/pull/38/commits/0696d02cb9368c31cfcd1230510a3f39ad3f44cd) was? I thought that it's better, but I'm just skipping a for statement , I'll change with that.